src/single-spa-angular1.js

```diff
@@ -68,7 +68,9 @@ function mount() {
 
 function unmount() {
 	return new Promise((resolve, reject) => {
-		let rootScope = angular.injector(['ng']).get('$rootScope');
+		let rootElement = angular.element(getContainerEl().childNodes[0]);
```

Comment:
This is querying the first element that angular mounted which should have the $rootScope.